### PR TITLE
docs - embedded analytics sdk

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Mention of SDK, link to existing docs.
- Updates static embedding parameters (downloads, background).
- Updates screenshots to reflect UI updates.
- Some peripheral cleanup.
- Adds quickstart with sample application.

### CODE REVIEW

**Review by albertoperdomo — CHANGES_REQUESTED** [review]
I left a few comments for what stood out the most.
In addition, I am surprised we go with JWT and Pro for the Quickstart. This makes it less accessible for people to start with the SDK.

We made it possible to use the SDK with API keys so anyone could try it out w/o having to get a license, an EE binary or have to implement JWT. I think having both makes sense. Or having API keys be the starting point and then JWT be the next, more advanced step.

File: docs/embedding/sdk/introduction.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - Official collections
+  - Subscriptions
+  - Alerts
+- The Metabase Embedding SDK doesn't support server-side rendering (SSR).
+- You can't embed multiple interactive dashboards on the same application page. If you need to embed multiple dashboards on the same application page, you can embed static dashboards.
+
+## Feedback
+
+- Email the team at [sdk-feedback@metabase.com](mailto:sdk-feedback@metabase.com).
+- Message the team on Slack. If you don't have a Slack channel set up, please reach out to us at [sdk-feedback@metabase.com](mailto:sdk-feedback@metabase.com).
```
[the remark this answers is not in the document]
Comment (albertoperdomo) [reply docs/embedding/sdk/introduction.md:77]: Actually, let's leave it for now.

**Comment by jeff-bruemmer** [comment]
> I left a few comments for what stood out the most. In addition, I am surprised we go with JWT and Pro for the Quickstart. This makes it less accessible for people to start with the SDK.
> 
> We made it possible to use the SDK with API keys so anyone could try it out w/o having to get a license, an EE binary or have to implement JWT. I think having both makes sense. Or having API keys be the starting point and then JWT be the next, more advanced step.

Mentioned on Slack, but commenting here for posterity; I haven't really started documenting the SDK yet; this is essentially a port of the existing docs. And there are some key pieces still in flux (e.g., ModifyQuestion) to handle.

I did the JWT quick start to take advantage of the sample app, and the setup is actually pretty easy, but we'll do an API keys quickstart too so people can try without a license. As we discussed a while back, we may want to pluck out that JWT setup later on so that it can be used in both interactive and SDK quickstarts, but I don't want to start optimizing until we have everything documented first.

File: docs/embedding/sdk/dashboards.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
+---
+title: "Embedded analytics SDK - dashboards"
+---
+
+## Embedded analytics SDK - Dashboards
+
+You can embed an interactive or static dashboard.
```
[the remark this answers is not in the document]
Comment (jeff-bruemmer) [reply docs/embedding/sdk/dashboards.md:7]: Just noting that it's likely we'll remove the "editable" question to simplify things.

File: docs/embedding/sdk/development.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
+---
+title: "Embedded analytics SDK - development
```
[the remark this answers is not in the document]
Comment (albertoperdomo) [reply docs/embedding/sdk/development.md:2]: I would keep this only on GH.

**Review by albertoperdomo — APPROVED** [review]

File: docs/embedding/sdk/version.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+# Embedded analytics SDK - versions
+
+{% include beta-blockquote.html %}
+
+{% include plans-blockquote.html feature="Embedded analytics SDK" %}
+
+The SDK version tracks with the Metabase version.
+
+So, for example, if you're on Metabase version `1.51.0`, you should use the SDK `1.51.0`.
```
Comment (jeff-bruemmer) [inline docs/embedding/sdk/version.md:13]:
```suggestion
So, for example, if you're on Metabase version `1.51.1`, you should use the SDK `1.51` or higher.
```

File: docs/embedding/sdk/version.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+# Embedded analytics SDK - versions
+
+{% include beta-blockquote.html %}
+
+{% include plans-blockquote.html feature="Embedded analytics SDK" %}
+
+The SDK version tracks with the Metabase version.
+
+So, for example, if you're on Metabase version `1.51.1`, you should use the SDK `1.51` or higher.
```
Comment (jeff-bruemmer) [inline docs/embedding/sdk/version.md:13]:
```suggestion
So, for example, if you're on Metabase version `1.51.1`, you should use the SDK `1.51`.
```

**Pull request merged**